pkg/client/unversioned/clientcmd/api/types.go

```diff
@@ -29,7 +29,11 @@ type Config struct {
 	// Legacy field from pkg/api/types.go TypeMeta.
 	// TODO(jlowdermilk): remove this after eliminating downstream dependencies.
 	Kind string `json:"kind,omitempty"`
-	// Version of the schema for this config object.
+	// DEPRECATED: APIVersion is the preferred api version for communicating with the kubernetes cluster (v1, v2, etc).
+	// Because a cluster can run multiple API groups and potentially multiple versions of each, it no longer makes sense to specify
+	// a single value for the cluster version.
+	// This field isnt really needed anyway, so we are deprecating it without replacement.
+	// It will be ignored if it is present.
```

Comment:
That makes this more of a removal.  If we're removing it, why aren't we simply removing the entire field?  This will look deceptive anywhere its touchable.